README.adoc

```diff
@@ -527,6 +530,31 @@ beforeStop:
     tail: 10
 ----
 
+=== CubeSpecificProperties
+
+In case you are using COMPOSE format, you cannot configure cube custom properties like `await` or `beforeStop`.
+You can use a property called `cubeSpecificProperties` allows you to add them in already defined containers.
+
+NOTE: This property only overrides custom properties defined by CUBE format.
+
+[source, xml]
+.arquillian.xml
+----
+<property name="cubeSpecificProperties">
+    tomcat:
+      image: tutum/tomcat:8.0
+      await:
+        strategy: polling
+      beforeStop:
+        - copy:
+            from: /test
+            to: /tmp";
+</property>
+----
+
+Previous snippet would override `await` strategy to polling and set a `beforeStop` event in cube container named `tomcat` defined in a previous docker-compose definition.
+Also notice that the `image` property here does nothing, since ony CUBE custom properties are overriden.
```

Comment:
You mean `image`. Yes I though the same but I found a way to enforce the message to reader that only custom properties are updated.